sdk/communication/azure-communication-rooms/src/samples/java/com/azure/communication/rooms/ReadmeSamples.java

```diff
@@ -173,30 +169,39 @@ public void addOrUpdateRoomParticipantsWithRoomId() {
         participantsToaddOrUpdate.add(participantToAdd); // Adding new participant to room
         participantsToaddOrUpdate.add(participant2); // Update participant from Consumer -> Attendee
 
-        RoomsClient roomsClient = createRoomsClientWithConnectionString();
-
         try {
             AddOrUpdateParticipantsResult addOrUpdateResult = roomsClient.addOrUpdateParticipants("<Room Id>", participantsToaddOrUpdate);
         } catch (RuntimeException ex) {
             System.out.println(ex);
         }
+        // END: readme-sample-addOrUpdateRoomParticipantsWithRoomId
     }
-    // END: readme-sample-addOrUpdateRoomParticipantsWithRoomId
 
-    // BEGIN: readme-sample-removeRoomParticipantsWithRoomId
     public void removeRoomParticipantsWithRoomId() {
+        RoomsClient roomsClient = createRoomsClientWithConnectionString();
+
+        // BEGIN: readme-sample-removeRoomParticipantsWithRoomId
         List<CommunicationIdentifier> participantsToRemove = new ArrayList<>();
 
         participantsToRemove.add(participant1.getCommunicationIdentifier());
         participantsToRemove.add(participant2.getCommunicationIdentifier());
 
-        RoomsClient roomsClient = createRoomsClientWithConnectionString();
-
         try {
             RemoveParticipantsResult removeResult = roomsClient.removeParticipants("<Room Id>", participantsToRemove);
         } catch (RuntimeException ex) {
             System.out.println(ex);
         }
+        // END: readme-sample-removeRoomParticipantsWithRoomId
+    }
+
+    public void listRoomParticipantsWithRoomId() {
+        RoomsClient roomsClient = createRoomsClientWithConnectionString();
+        // BEGIN: readme-sample-listRoomParticipantsWithRoomId
+        try {
+            PagedIterable<RoomParticipant> allParticipants = roomsClient.listParticipants("<Room Id>");
```

Comment:
nit: it would be nice to have a loop here that prints an indiviual participant.

Something like this:
  for (RoomParticipant participant : participants) {
         System.out.println(participant.getCommunicationIdentifier().getRawId() + " (" + participant.getRole() + ")");
     }

Reply:
Great suggestion, added!